src/content/pages/speakers-dinner.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: Speaker's Dinner
+url: /speakers-dinner
+subtitle:
+  We're happy to invite all speakers to our special Speaker's Dinner!
+---
+
+# Speaker's Dinner
+We are excited to announce a special Speaker's Dinner! This exclusive event is for speakers only, providing an opportunity to meet and mingle before the main conference days.
+
+The dinner will feature multiple courses, and you can choose from a variety of alcoholic and non-alcoholic beverages. We also offer vegetarian, vegan, and gluten-free options to accommodate all dietary preferences.
```

Comment:
```suggestion
The dinner will feature multiple courses and you can choose from a variety of alcoholic and non-alcoholic beverages. We also offer vegetarian, vegan, and gluten-free options to accommodate all dietary preferences.
```
I believe that food and drinks should be kept together and therefore the separating "," is not the right choice?

Reply:
ChatGPT says `,` makes it more readable. 🤷🏻‍♀️ 

![image](https://github.com/EuroPython/website/assets/38294198/c55a4d90-428b-4d5a-87a6-12bfa56dca6c)

Reply:
> vegetarian, vegan, and gluten-free options to accommodate all dietary preferences.

this sentence suggests vegetarian, vegan and gf are all the dietary preferences. 

If you don't have any other option, perhaps something like this is clearer: "We also offer  vegetarian, vegan, and gluten-free options to cater to various dietary preferences."

If you have other options to choose from, maybe "We also offer a variety of options to accommodate different dietary preferences, including vegetarian, vegan, and gluten-free."?

Reply:
Thanks for the suggestion, I've included the second sentence.